### PR TITLE
Feature/display of planning data

### PR DESCRIPTION
**Pull request by matkoniecz**
replaces #897 due to change to permission levels

### CODE REVIEW

**Comment by polly64** [comment]
Hi @edwardchalstrey1  @fallipalaiologou 
Hi @matkoniecz
please can you also

- take old questions down  
- change link in top box  to the https://github.com/colouring-cities/manual/wiki/G1.-Data-capture-methods
- Change disclaimer text based on my earlier comments thanks
-remove the title 'Planning application status' but keep the same title where it exists  with the arrow next to just put green box within concertina 
- Give a planned date for activation of the out of green box crowdsourcing questions- Can you send these to ed today? i.e. at  https://github.com/colouring-cities/colouring-london/issues/884. 
he is leaving and there will be a delay before Mike starts and he will have a lot of other priorities 

- Also a lot of the entries are yellow which means appeals but these are normally rare- something wrong here could you check thanks

**Comment by matkoniecz** [comment]
@tomalrussell Review is also welcome

File: etl/planning_data/obtain_livestream_data_and_load_into_database.py [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        '_source': {
+            'excludes': [],
+        },
+        'query': {
+            'bool': {
+                'must': [
+                    {
+                        'range': {
+                            'valid_date': {
+                                'gte': '01/01/1021',
```
Comment (tomalrussell) [inline etl/planning_data/obtain_livestream_data_and_load_into_database.py:139]: 1021 is a long time ago - was that meant to be 2021?

File: etl/planning_data/obtain_livestream_data_and_load_into_database.py [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+import datetime
+import os
+import requests
+import psycopg2
+import address_data
+
+def main():
+    connection = get_connection()
+    cursor = connection.cursor()
+    cursor.execute("TRUNCATE planning_data")
```
Comment (tomalrussell) [inline etl/planning_data/obtain_livestream_data_and_load_into_database.py:11]: Is this meant to be re-run regularly? Could we do something more conservative than `TRUNCATE` and reload all? It seems quite risky to drop everything and hope the bulk reload works okay every time.

File: etl/planning_data/obtain_livestream_data_and_load_into_database.py (hunk 1 above)
Comment (matkoniecz) [reply etl/planning_data/obtain_livestream_data_and_load_into_database.py:139]: In this case it was intended to cover all data (as one point I tested it and it needed some minimum date).

There is scattering of some old data like something from 1968, so in theory they may add older ones. Not sure when the first equivalent of planning permission appeared in London, so I set date that would catch everything.

Comment (matkoniecz) [reply etl/planning_data/obtain_livestream_data_and_load_into_database.py:139]: Though I will add comment explaining situation and retry is this weird filter avoidable.

**Comment by mdsimpson42** [comment]
I've checked it out and it looks fine to me.

**Comment by matkoniecz** [comment]
@tomalrussell Is it safe (and a good idea) to store in `geometries` table geometries not representing buildings?

It seems to be necessary to display borough labels requested by @polly64 - because different boroughs have drastic differences in data quality they provide.

And to get decent labels I render them using mapnik, with one more tile layer, as that seems beyond what GeoJSON layer can do.

It would be also possible to put them in a completely separate table, without putting this data into geometries table.

main commit is in https://github.com/colouring-cities/colouring-london/pull/898/commits/c4933fdc11118e78d19060584280304485c613c0 (with some later tweaking of label styling)

**Comment by matkoniecz** [comment]
Also, I keep wondering what is the best place to store geometry migration data - I am not really happy about https://github.com/colouring-cities/colouring-london/pull/898/commits/c6831b9f794e698ca95acec15f1e90760ac2aaa9 putting them in migration file, but it would be nice to have them somewhere (it was already a bit tricky to reconstruct them for me, now just few days after initial coding and being the author as I deploy them to staging)

**Comment by matkoniecz** [comment]
@mdsimpson42 This one should be overall in mergeable state. There are some minor upcoming tweaks on TODO list, but nothing broken/fundamental.

(though I am sure that review would find things that could be done better)
